interfaces/builtin/audio_playback.go

```diff
@@ -140,13 +142,24 @@ func (iface *audioPlaybackInterface) StaticInfo() interfaces.StaticInfo {
 	return interfaces.StaticInfo{
 		Summary:              audioPlaybackSummary,
 		ImplicitOnClassic:    true,
-		ImplicitOnCore:       true,
 		BaseDeclarationSlots: audioPlaybackBaseDeclarationSlots,
 	}
 }
 
 func (iface *audioPlaybackInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
 	spec.AddSnippet(audioPlaybackConnectedPlugAppArmor)
+
+	if slot.Snap().SnapName() != "snapd" {
+		// This enables audio in Core Desktop, by giving access to the
+		// sockets inside the snap's own folder.
+		instanceName := slot.Snap().InstanceName()
+		coreAppArmor := fmt.Sprintf(`
+owner /run/user/[0-9]*/snap.%s/pulse/ rw,
```

Comment:
I know you've copied this from the existing rules, but it seems particularly odd to give write access to this directory. Do things break with just read access? If not, it might make sense to drop it from the equivalent rule in `audioPlaybackConnectedPlugAppArmorDesktop`.

Reply:
Good point... I'll test it.

Reply:
It does work. Should I also remove it from `audioPlaybackConnectedPlugAppArmor`?

Reply:
I think so. As is, it seems like it'd let a plug snap replace the pulseaudio socket, and capture other apps' audio, which doesn't seem great.

Reply:
Done, and also in the pulseaudio interface.